packages/grid/test/filtering.common.js

```diff
@@ -51,7 +49,7 @@ describe('filter', () => {
 
   beforeEach(async () => {
     filterWrapper = fixtureSync('<filter-wrapper></filter-wrapper>');
-    await filterWrapper.updateComplete;
+    await nextRender();
```

Comment:
This timing change is needed to ensure that all the elements are fully rendered:

`filter-wrapper` -> `vaadin-grid-filter` -> `vaadin-text-field` -> slotted `<input>`